bridge/src/main/java/com/livefront/bridge/BridgeDelegate.java

```diff
@@ -41,14 +46,18 @@
                    @NonNull SavedStateHandler savedStateHandler) {
         mSharedPreferences = context.getSharedPreferences(TAG, Context.MODE_PRIVATE);
         mSavedStateHandler = savedStateHandler;
+        registerForLifecycleEvents(context);
     }
 
     void clear(@NonNull Object target) {
+        if (!mIsClearAllowed) {
```

Comment:
Right, no way to reliably do it right now that I can think of. It was either this or just bump the min SDK up.